### DISCUSSION TITLE
Problem with local nugets

### DISCUSSION THREAD

**Discussion by atari-monk** [bilal-fazlani/commanddotnet#477] [General]
Can you please give me a comment on some issue ?
I upgraded nugets to net7. Also refactored one bad micro lib.
I try to run Better.Console.Tables.TestApp to test.
When i add reference to projs 
Include="..\CommandDotNet.Helper\CommandDotNet.Helper.csproj
Include="..\CommandDotNet.Unity.Helper\CommandDotNet.Unity.Helper.csproj"
it works fine.
If i make local nuget out of them and referance them as nugets, i get error.
Have you seen that ?

**Comment by drewburlingame** [comment]
I haven't seen that or used that workflow in a while.

**Reply by atari-monk** [reply]
Oh what a relief.
I wasted so much time on this.
I was probably wrong doing this, on so many many levels.
As you pointed out. 
(Crazy amount of code that does nothing but obscure)
I want to fix that and never do that.
Wonder if thats possible or if my thinking is damaged that way. 
But i am happy it started to work.
Aparenlty nuget was not updated.
```
cd nuget proj path
dotnet nuget locals all --clear
dotnet nuget locals temp -c
dotnet nuget locals global-packages -c
dotnet restore
dotnet pack -c Release -o C:\atari-monk\nugets
```
After doing this cmds from some blog it updated.
I will be able to waste some more time :)
